client/my-sites/backup/backup-contents-page/index.tsx

```diff
@@ -99,11 +102,41 @@ const BackupContentsPage: FunctionComponent< OwnProps > = ( { rewindId, siteId }
 							</div>
 						</div>
 						<div className="status-card__title">{ displayDate }</div>
-						{ fileBrowserState.getCheckList().totalItems === 0 && (
-							<Spacer marginBottom={ 2 }>
+						<Spacer marginBottom={ 2 }>
+							{ fileBrowserState.getCheckList().totalItems === 0 ? (
 								<ActionButtons isMultiSite={ isMultiSite } rewindId={ rewindId.toString() } />
-							</Spacer>
-						) }
+							) : (
+								<ButtonStack justify="flex-start">
+									<Button
+										__next40pxDefaultSize
+										onClick={ handleRequestGranularDownload }
+										variant="secondary"
+									>
+										{ translate( 'Download selected files' ) }
+									</Button>
+									<Button
+										__next40pxDefaultSize
+										onClick={ () => {
+											dispatch(
+												recordTracksEvent(
+													'calypso_jetpack_backup_browser_restore_multiple_files',
+													{
+														...( hasCredentials !== undefined && {
+															has_credentials: hasCredentials,
+														} ),
+													}
+												)
+											);
+											handleRequestGranularRestore( siteSlug, rewindId );
+										} }
```

Comment:
There's too much nesting in this `onClick` prop. We can increase the readability considerably by defining this outside of the component declaration and referencing it like `onClick={ handleOnClick }`.

Reply:
Oh, I left that mess there. Let me fix it before merging :D